include/cupla/cudaToCupla/driverTypes.hpp

```diff
@@ -95,7 +95,14 @@
   static_cast<uint3>(                                                \
       ::alpaka::workdiv::getWorkDiv<::alpaka::Thread, ::alpaka::Elems>(acc))
 
-// atomic functions
+/** Atomic functions
+ *
+ * Compared to their CUDA counterparts, these functions take an additional last
+ * parameter to denote atomicity (synchronization) level. This parameter is
+ * of type ::alpaka::hierarchy::{Grids|Blocks|Threads}. Grids corresponds
+ * to atomicity between different kernels, Blocks - to different blocks
+ * in the same kernel, Threads - to threads of the same block.
+ */
```

Comment:
Use [doxygen groups](http://doxygen.10944.n7.nabble.com/Doxygen-for-multiple-defines-td493.html) here :)

Reply:
Good point, fixed.